src/main/asciidoc/api/chapter.adoc

```diff
@@ -34,6 +34,8 @@ ArcadeDB provides <<HTTP-API,HTTP API>> to interface to the server from a remote
 | Java | Bundled with the project. Look at `Remote Database` class | Apache 2
 | CHICKEN Scheme | http://wiki.call-cc.org/eggref/5/arcadedb | zlib-acknowledgement
 | .NET | https://github.com/tetious/ArcadeDb.Client | Apache 2
+| Python | https://github.com/adaros92/arcadedb-python-driver | Apache 2
+| Python | https://github.com/ExtReMLapin/pyarcade | Apache 2
```

Comment:
From the readme "Based on https://github.com/adaros92/arcadedb-python-driver but reworked and with bugfixes" but I don't see it's a fork. If one is based on the other one, we should probably pick the most updated (pyarcade)?

Reply:
I included both, because as you state the "newer" states it is reworked and has bug fixes, but the other "older" is listed at PyPi, see https://pypi.org/project/pyarcade/ , which is likely easier to install. I think the fork that was started off is here https://github.com/ExtReMLapin/arcadedb-python-driver .

Reply:
I see the original author yesterday merged the PR by @ExtReMLapin

Reply:
Yes but I only pushed a fix on his branch, yet my repo contains a much larger rewrite than his lib

Reply:
I see it's not technically a fork. What are the differences?

Reply:
If we're comaring adaros's work to mine [here](https://github.com/ExtReMLapin/pyarcade) (not a fork)

I removed the overengineered code that just sounded silly, for example [instanciating a class that just contains two fields and call a method that turns it back into a dict](https://github.com/adaros92/arcadedb-python-driver/blob/bf98551b496ffe1cb36794576339ab61d6f8eca9/pyarcade/dao/database.py#L39)

It might be a convention, but I just find this kind of thing silly.


Original repo Database class : 

* class DatabaseDao
   * method _db_exists
   * method get_db
   * method create_db
  
Mine : 

* class DatabaseDao
    * class IsolationLevel
        * const READ_COMMITTED
        * const REPEATABLE_READ
    * class method exists
    * class method create
    * class method delete
    * class method list_databases
    * method query
    * method begin_transaction
    * method commit_transaction
    * method rollback_transaction
    * method __repr__

With my repo, queries are ran on the Database object, not on the client object (where you have to specify the full endpoint that contains for example the database name)

Also when creating a client it auto checks at instanciation if the password is valid.
I also implemented a primitive but (imho) good enough exception handling

A query actually returns a list if needed intead of the json object returned from the API with the server informations.